js/login.js

```diff
@@ -0,0 +1,68 @@
+import { browserHistory } from 'react-router';
+import Service from './service.js';
+import UserData from './user-data.js';
+
+export default {
+  clearLoggedInParam(locationObj) {
+    let query = locationObj.query;
+
+    delete query.loggedin;
```

Comment:
Just because `delete` is a really expensive operation, let's wrap this as
```
if (query.loggedin) {
  delete query.loggedin;

  browserHistory.replace({
    pathname: locationObj.pathname,
    query: query
  });
}
```
While there *should* be a `loggedin` query param, we can never be 100% certain. The internet has been known to break.